Kickstarter-iOS/Features/ReportProject/ReportProjectInfoView.swift

```diff
@@ -0,0 +1,99 @@
+import Library
+import SwiftUI
+
+enum ReportProjectHyperLinkType: String, CaseIterable {
+  case prohibitedItems
+  case communityGuidelines
+
+  func stringLiteral() -> String {
+    switch self {
+    case .prohibitedItems:
+      return Strings.Prohibited_items()
+    case .communityGuidelines:
+      return "community guidelines"
+    }
+  }
+}
+
+@available(iOS 15, *)
+struct ReportProjectInfoView: View {
+  var body: some View {
+    NavigationView {
+      VStack(alignment: .leading, spacing: 10) {
+        Text(Strings.Report_this_project())
+          .font(Font(UIFont.ksr_title1()))
+          .bold()
+          .padding()
```

Comment:
I don't think this should be a separate nav title under the navigation bar. I think the back button and the title should be on the same navigation bar. It should look like "Updates" or "Comments".